README_libf3d.md

```diff
@@ -0,0 +1,224 @@
+
+## Render Options
+Options|Default|Type|Description|F3D option|Trigger
+------|------|------|------|------|------
+render.effect.depth-peeling|false|bool|Enable *depth peeling*. This is a technique used to correctly render translucent objects.|--depth-peeling|render
+render.effect.fxaa|false|bool|Enable *Fast Approximate Anti-Aliasing*. This technique is used to reduce aliasing.|--fxaa|render
+render.effect.ssao|false|bool|Enable *Screen-Space Ambient Occlusion*. This is a technique used to improve the depth perception of the object.|--ssao|render
+render.effect.tone-mapping|false|bool|Enable generic filmic *Tone Mapping Pass*. This technique is used to map colors properly to the monitor colors.|--tone-mapping|render
+render.line-width|1.0|double|Set the *width* of lines when showing edges.|--line-width|render
+render.show-edges|false|bool|Show the *cell edges*|--edges|render
+render.point-size|10.0|double|Set the *size* of points when showing vertices and point sprites.|--point-size|render
+render.point-sprites.enabled|false|bool|Show sphere *points sprites* instead of the geometry.|--point-sprites|render
+render.volume.enabled|false|bool|Enable *volume rendering*. It is only available for 3D image data (vti, dcm, nrrd, mhd files) and will display nothing with other default scene formats.|--volume|render
+render.volume.inverse|false|bool|Inverse the linear opacity function.|--inverse|render
+render.raytracing.denoise|false|bool|*Denoise* the raytracing rendering.|--denoise|render
+render.raytracing.enable|false|bool|Enable *raytracing*. Requires the raytracing module to be enabled.|--raytracing|render
+render.raytracing.samples|5|int|The number of *samples per pixel*.|--samples|render
+
+## UI Options
+Options|Default|Type|Description|F3D option|Trigger
+------|------|------|------|------|------
+ui.bar|false|bool|Show *scalar bar* of the coloring by data array.|--bar|render
+ui.cheatsheet|false|bool|Show a interactor cheatsheet|render
+ui.filename|false|bool|Display the *name of the file*.|--filename|render
+ui.font-file||string|Use the provided FreeType compatible font file to display text.<br>Can be useful to display non-ASCII filenames.|--font-file|render
+ui.fps|false|bool|Display a *frame per second counter*.|--fps|render
+ui.loader-progress|false|bool|Show a *progress bar* when loading the file.|--progress|load
+ui.metadata|false|bool|Display the *metadata*.|--metadata|render
+
+## Window Options TODO remove
+Options|Default|Type|Description|F3D option|Trigger
+------|------|------|------|------|------
+window.fullscreen|false|bool|Display in fullscreen.|--fullscreen|render
+
+# Python Bindings
+
+If the python bindings are generated, the libf3d can be used directly from python.
+Make sure to set `PYTHONPATH` to path where the python module is built.
+Here is an example showing how to use libf3d python bindings:
+
+```python
+import f3d
+
+eng = f3d.engine()
+eng
+ .getOptions().set("model.scivis.array-name", "Normals")
```

Comment:
do you have to repeat the `.getOptions()` for python? Can't you just chain the `.set` calls?